proto/proto-backplane-grpc/src/main/java/io/deephaven/proto/util/SharedTicketHelper.java

```diff
@@ -0,0 +1,36 @@
+//
+// Copyright (c) 2016-2024 Deephaven Data Labs and Patent Pending
+//
+package io.deephaven.proto.util;
+
+import com.google.protobuf.ByteString;
+import org.apache.commons.codec.binary.Hex;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.List;
+
+public class SharedTicketHelper {
+    public static final char TICKET_PREFIX = 'h';
+    public static final String FLIGHT_DESCRIPTOR_ROUTE = "shared";
+
+    /**
+     * Convenience method to create the flight descriptor path for the given shared variable identifier.
+     *
+     * @param variableId the variable identifier
+     * @return the path
+     */
+    public static List<String> nameToPath(ByteString variableId) {
```

Comment:
I also wonder if `ByteString` is the API we want here. Alternatives could be `byte[], int offset, int len`, `byte[]`, or `ByteBuffer`?